### PR TITLE
Login Screen functionality

### PR DESCRIPTION
**Pull request by JohnThirlaway1**
connects to #25

### CODE REVIEW

File: app/components/TeamCalendar.js [hunk 1]
```diff
@@ -24,6 +25,7 @@ var TeamCalendar = React.createClass({
   render: function () {
     return (
       <div id='teamCalendar'>
+      <Navbar />
```
Comment (AlexJYoungman) [inline app/components/TeamCalendar.js]: Indentation

File: app/components/login.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var InputPassword = require('react-ux-password-field');
+
+var Login = React.createClass({
+  getInitialState: function() {
```
Comment (AlexJYoungman) [inline app/components/login.js]: Space between function and brackets.
`function () {`

Comment (AlexJYoungman) [inline app/components/login.js]: Makre sure to double check through the files and add the space where needed

File: app/components/login.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+var Results = React.createClass({
+  getInitialState: function() {
+    return { showAltLogin: false };
+  },
+  onClick: function() {
+    this.setState({ showAltLogin: true });
+
+  },
+  function() { return {query: ''} },
```
Comment (AlexJYoungman) [inline app/components/login.js]: Add spaces inside object.
`{ query: '' }`

Comment (AlexJYoungman) [inline app/components/login.js]: Again, just check through the file and fix these.

File: app/components/MyCalendar.js [hunk 4]
```diff
@@ -224,6 +225,7 @@ var MyCalendar = React.createClass({
   render: function () {
     return (
       <div id='myCalendar'>
+      <Navbar />
```
Comment (TheGarrett) [inline app/components/MyCalendar.js]: indentation

File: app/components/Users.js [hunk 5]
```diff
@@ -1,5 +1,10 @@
 var React = require('react');
+<<<<<<< HEAD
```
Comment (TheGarrett) [inline app/components/Users.js]: remove these

**Pull request merged**